command/start.go

```diff
@@ -89,6 +94,13 @@ func (c *startCommand) startFlags() *flag.FlagSet {
 	flags.BoolVar(&autocompleteUninstall, flagAutocompleteUninstall, false, "Uninstall the autocomplete")
 	c.autocompleteUninstall = &autocompleteUninstall
 
+	// Sets that we are using start as the because CTS invoked without a command. Not printed with -h, -help
+	flags.BoolVar(&isDeprecatedStartup, flagDeprecatedStartUp, false,
+		"Use only when running CTS without a command. "+
```

Comment:
In an earlier comment, mentioned how this reads as though is a flag that users pass in. Wasn't sure what your thoughts were on updating it to be "Automatically set by CTS only when running..."?